examples/load-tpanalyze-package.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+description: example script of how to load the tpanalyze package
+created: 2018-02-21
+author: Ed Nykaza
+license: BSD-2-Clause
+"""
+
+
+# PLEASE NOTE: THESE 10 LINES ARE NEEDED TO LOAD THE TPANALYZE PACKAGE
+# ALSO NOTE: THIS PACKAGE IS STILL IN DEVELOPMENT
+import sys
+import os
+cwd = os.getcwd()
+# NOTE: if you renamed the root directory of the data-analytics repository,
+# enter that name here:
+nameDataAnalyticsRepository = "data-analytics"
+packagePath = cwd[:(cwd.find(nameDataAnalyticsRepository) +
+                    len(nameDataAnalyticsRepository) + 1)]
+sys.path.append(packagePath)
+sys.path.append(os.path.join(packagePath, "tpanalyze"))
+import tpanalyze as tp
+import pandas as pd
+
+
+# %% load in example data with the tpanalyze package
+dataPath = os.path.join("..", "example-data", "example-from-j-jellyfish.csv")
```

Comment:
The ".." command brings you to the relative parent directory of whatever your current working directory is. If your current working directory is C:/data-analytics then the dataPath will be "C:/example-data/example-from-j-jellyfish.csv" which does not exist.

The packagePath is guaranteed to be the data-analytics main folder, so I might recommend:
dataPath = os.path.join(packagePath, "example-data", "example-from-j-jellyfish.csv")

Reply:
done